openvdb/openvdb/Types.h

```diff
@@ -9,7 +9,7 @@
 #include "TypeList.h" // backwards compat
 
 #ifdef OPENVDB_USE_IMATH_HALF
-#include <OpenEXR/half.h>
+#include <half.h>
```

Comment:
Is this correct? As far as I remember the path to half was `include/OpenEXR/half.h`

Reply:
Since Half used to be a separate library, under Half/half, and is now in Imath, and is not part of OpenEXR, the forward looking approach is to refer to half.h directly, through having it discoverable in the include search paths.

Reply:
At the moment our FindOpenEXR module brings in both `-I/EXR_ROOT/include/OpenEXR -I/EXR_ROOT/include` because of the way IlmBase and OpenEXR used to be installed separately. I really don't like doing `-I/IMATH_ROOT/include/Imath` and then bringing in half with `#include <half.h>`, but seems the alternative is to introduce a header define (https://github.com/AcademySoftwareFoundation/Imath/blob/master/docs/PortingGuide2-3.md#accommodating-openexrimath-3x-or-openexr-2x-1) as there's no filepath alias... Does the Imath config module bring in both include paths? i.e. if I built with `find_package(Imath CONFIG) + target_link_libraries(myproj Imath)` does that bring in the right include paths such that writing `#include <half.h>` works? If not, we'll have to go with the define branch option.

Reply:
I changed this to do a conditional include of "OpenEXR/half.h" or "Imath/half.h" based on the Imath_VERSION, now included in version.h. I think this is the best solution.